packages/IMAPClient-Core.package/ICEndPoint.class/class/checkValidConfiguration..st

```diff
@@ -0,0 +1,8 @@
+storage
+checkValidConfiguration: aDataDictionary
+
+	^ (aDataDictionary includesKey: 'username') and:
+	(aDataDictionary includesKey: 'port') and:
+	(aDataDictionary includesKey: 'address') and:
+	(aDataDictionary includesKey: 'name') and:
+	(aDataDictionary includesKey: 'ssl')
```

Comment:
Shouldn't these be blocks `[]` instead of parens `()`?

Reply:
They could be, but don't have to. With `()` the contents get evaluated to `true`/`false` before `and:` is called. Since the calls to `includesKey:` don't depend on any call-order it does not matter

Reply:
But how does Squeak work here then. Does it check for the argument to `and:` to be a block or a boolean? Anyway I'd still prefer to have it be `[]`, for style reasons.